input_box.py

```diff
@@ -0,0 +1,26 @@
+from PyQt5 import QtWidgets, QtCore
+from text_widget import Textbox
+from PyQt5.QtWidgets import QApplication, QMainWindow, QLineEdit, QLabel
+from PyQt5.QtGui import *
+import sys
+class Inputbox(QtWidgets.QLineEdit):
+    def __init__(self,parent=None):
+        super(Inputbox,self).__init__(parent)
+        self.parent=parent
+        self.initUI()
+
+
+    def initUI(self):
+        self.resize(200, 50)
+        self.move(155, 200)
+
+

```

Comment:
When the return key is pressed call `do_something`. In `do_something` you're going to tell the parent class (i.e., `Game`) to check if it needs to show the WPM by calling `self.parent.test()`. Then, you're going to do anything else that needs to be done to the InputBox. For example, resetting the textbox (`self.setText('')`)

```suggestion
def do_something(self):
    self.parent.test()
    self.setText('')
```